Zappy_IA_Report/DTC/Code/protocole.py

```
@@ -0,0 +1,210 @@
+        return
+
+    if client["commandes"]:
+        command = client["commandes"].pop(0)
+        handle_commande(client, command, message)
+
+        if command != "Dead":
+            ml_agent.strategy(client)
+
+
+def execute_command(client, commande, args) -> None:
+    global allowed_commands
```

Flake8 (7.2.0)

[error] `global allowed_commands` is unused: name is never assigned in scope

(F824)